Formula/luvit.rb

```diff
@@ -30,6 +30,14 @@ class Luvit < Formula
     url "https://github.com/luvit/lit.git",
         tag:      "3.8.5",
         revision: "84fc5d729f1088b3b93bc9a55d1f7a245bca861d"
+
+    livecheck do
+      url "https://raw.githubusercontent.com/luvit/luvit/master/Makefile"
+      regex(/(LIT_VERSION=)+(\d+(?:\.\d+)+)/i)
```

Comment:
We want the version our release depends on, not master

Reply:
Yes, it was mentioned by @nandahkrishna as well.

However, we need access to the Formula’s `livecheck` result inside the resources’ `livecheck` block (perhaps by way of a helper method), so that we can use the Formula’s latest URL and check the right _file_/_webpage_ for resource versions.

Reply:
I definitely agree that we need to be checking a file from a release version (not the `master` branch) but using the `Makefile` for the current formula version would lead to a situation where livecheck returns a new version for the formula but the resource versions are for the _current_ release. As described in https://github.com/Homebrew/homebrew-core/pull/112155#issuecomment-1355090824, we need to check the file from the latest version reported by livecheck for the main formula check and that's what Mohammad is describing above.

This isn't possible at the moment because the latest version from livecheck is determined at runtime, not when the formula is parsed. We've discussed this issue internally and Nanda's working on Homebrew/brew changes to make it possible to reference the latest version from the main formula check in a resource `livecheck` block and should have some work up soon. In the interm time, we can't create an appropriate `livecheck` block for this resource until then.